### ISSUE TITLE
Clear Cache modal, accessibility and focus improvements

### ISSUE THREAD

**Issue by 7ruth** [DecipherNow/gm-fabric-dashboard#1121]
## Expected Behavior

As focus shifts to this modal when you click into it, the outline should not be highlighted. 
As the focus shifts, the modal should be announced in the screen reader.

## Actual Behavior

The shift in focus is not announced in the voiceOver screen reader
Clicking with the mouse anywhere in the modal produces an outline around the whole modal

## Steps to Reproduce the Problem

  1. Navigate to settings view and click clear cache button
  
## Specifications

  - Version: 1.0.1
  - Platform: fabric
